torch/csrc/jit/script/compiler.cpp

```diff
@@ -2266,6 +2266,12 @@ struct to_ir {
     } else {
       auto inputs = getNamedValues(apply.inputs(), true);
       auto attributes = emitAttributes(apply.attributes());
+      if (sv->kind() == "python value of type \'builtin_function_or_method\'") {
```

Comment:
This error message probably should go in PythonValue::call not here. `kind()` is for error message reporting only and shouldn't be used to decide behavior.